website/src/controllers/api-v3/groups.js

```diff
@@ -563,15 +520,25 @@ api.inviteToGroup = {
     let uuids = req.body.uuids;
     let emails = req.body.emails;
 
-    if (uuids && emails) { // TODO fix this, low priority, allow for inviting by both at the same time
-      throw new BadRequest(res.t('canOnlyInviteEmailUuid'));
-    } else if (Array.isArray(uuids)) {
-      // return _inviteByUUIDs(uuids, group, user, req, res, next);
-    } else if (Array.isArray(emails)) {
-      // return _inviteByEmails(emails, group, user, req, res, next);
-    } else {
+    if (!Array.isArray(uuids) && !Array.isArray(emails)) {
       throw new BadRequest(res.t('canOnlyInviteEmailUuid'));
     }
+
+    let results = [];
+
+    if (Array.isArray(uuids)) {
+      let uuidInvites = uuids.map((uuid) => _inviteByUUID(uuid, group, user, req, res));
+      let uuidResults = await Q.all(uuidInvites);
+      results.push(uuidResults);
+    }
+
+    if (Array.isArray(emails)) {
+      let emailInvites = emails.map((invite) => _inviteByEmail(invite, group, user, req, res));
+      let emailResults = await Q.all(emailInvites);
+      results.push(emailResults);
+    }
+
+    res.respond(200, {});
```

Comment:
@TheHollidayInn regarding this one it would be very useful to return the invitations that are stored on the users so we can update the UI without a new request